test/integration/docker-compose-rust.yml

```diff
@@ -2,7 +2,7 @@ version: '3.8'
 
 services:
   testserver:
-    image: grcevski/tests:greeting-rust${TESTSERVER_IMAGE_SUFFIX}
+    image: ghcr.io/grafana/beyla/test/greeting-rust${TESTSERVER_IMAGE_SUFFIX}/0.0.2
```

Comment:
That makes me remind that we should work on an image repo for this kind of test images. Also including ARM64 images.

Reply:
Ah yes! Let me open an issue. Thanks for the reminder.

Reply:
https://github.com/grafana/beyla/issues/308 Thanks again!